keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -449,30 +472,30 @@ status:
   - type: Conflict
     status: "True"
     lastTransitionTime: "2020-03-30T01:33:55Z"
-    message: "Conflicting type. Using \"ClusterSetIP\" from oldest service export in \"cluster-1\". 2/5 clusters disagree."
+    message: "Conflicting type. Using \"MultiClusterIP\" from oldest service export in \"cluster-1\". 2/5 clusters disagree."
 ```
 
 To export a service, a `ServiceExport` should be created within the cluster and
 namespace that the service resides in, name-mapped to the service for export -
 that is, they reference the `Service` with the same name as the export. If
-multiple clusters within the clusterset have `ServiceExports` with the same
-name and namespace, these will be considered the same service and will be
-combined at the clusterset level.
+multiple clusters associated with a multicluster namespace have 
+`ServiceExports` with the same name in that namespace, these will be considered the same service and will be combined at the multicluster namespace level, subject to implementation-specific policy.
 
 _Note: A `Service` without a corresponding `ServiceExport` in its local cluster
 will not be exported even if other clusters are exporting a `Service` with the
 same namespaced name._
 
-This requires that within a clusterset, a given namespace is governed by a
-single authority across all clusters. It is that authority’s responsibility to
-ensure that a name is shared by multiple services within the namespace if and
-only if they are instances of the same service.
+This requires that the exporting of services within a multicluster namespace
+be governed by a single authority across all clusters.  This may be achieved in different ways:
+- For example, in the case of a clusterset model with high-trust amongst clusters, an authority would ensure that the namespaced-name of a service is used across multiple clusters if and only if they are instances of the same service.
+- In the case of a federated, non-clusterset model, with low-trust between clusters, the implementation of this API must have outside knowledge of cluster-namespace ownership and use that to validate `ServiceExport`s being reconciled.
```

Comment:
My preference would be to avoid use of the term 'federation' here, it is nearly universally confusing in practice, especially given the presence of projects with that name in this community.